### PR TITLE
Allow some valid words

### PR DESCRIPTION
**Pull request by not-my-profile**
Stop correcting the following valid words:

* simulative
* pervious
* perviously
* intension
* intensional
* intensionally
* unuseful

### CODE REVIEW

File: crates/typos-dict/assets/allowed.csv [hunk 1]
```diff
@@ -6,3 +6,10 @@ deques,noun
 dequeues,verb
 ons,so `add-ons` works
 accreting,verb of accrete
+sortings,plural of sorting
```
Comment (epage) [inline crates/typos-dict/assets/allowed.csv:9]: Can you provide a source for this?

File: crates/typos-dict/assets/allowed.csv [hunk 2]
```diff
@@ -6,3 +6,10 @@ deques,noun
...
+simulative,adjective
+pervious,adjective
+perviously,adverb
```
Comment (epage) [inline crates/typos-dict/assets/allowed.csv:12]: Source: https://www.dictionary.com/browse/pervious

File: crates/typos-dict/assets/allowed.csv (hunk 1 above)
Comment (not-my-profile) [reply crates/typos-dict/assets/allowed.csv:9]: Merriam-Webster describes the 5th meaning of the "sorting" noun as:

> an instance of sorting

Multiple instances of sorting would thus be "sortings". See [Google Books for examples of "sortings"](https://www.google.com/search?q=%22sortings%22&tbm=bks) (currently yields 22.600 results). The English Wiktionary also has [a page](https://en.wiktionary.org/wiki/sortings) (though that probably doesn't mean much).

Comment (not-my-profile) [reply crates/typos-dict/assets/allowed.csv:9]: Ah, right ... good catch! I'll split this off.

Comment (not-my-profile) [reply crates/typos-dict/assets/allowed.csv]: I filed #752 for the further discussion on "sortings".

**Comment by not-my-profile** [comment]
Removed `unuseful,useless` since unuseful is a proper word [according to Merriam Webster](https://www.merriam-webster.com/dictionary/unuseful) and substituting one synonym for another seems awfully opinionated.

**Comment by not-my-profile** [comment]
Converted to a draft because the tests currently fail because of:

```
intensionaly,intentionally,intensionally
```

(the order isn't preserved because of #753 ... once #765 has been merged, I'll rebase this PR)

**Comment by szepeviktor** [comment]
@epage Oh no! People type `pervious` every day.
How to configure Typos to keep spotting this typo?

**Pull request merged**